2022-finalproject/README.md

```diff
@@ -1,44 +1,14 @@
-# Final Project
+# 收获
 
-- 本次大作业的文件除特殊要求外都应使用合理的英文名称命名
-- 通过 pull request 的方式提交到[课程仓库](https://github.com/gdut-dynamic-x/missing-semester)，具体位置为 `2022-finalproject` 文件夹下以你 GitHub ID 命名的文件夹
+从学习这个项目以来，我收获到了
+c+＋的基础学习
+linux 命令行的基础语法
+bash脚本基础
+vim编辑器的使用
+git入门语法
 
-## Pull Request
+作为刚入门的一个纯小白，甚至连Windows都弄不明白，就加入到了Ubuntu系统的学习，通过B站相关网课学习对应的内容，以及每学习一个内容，我都会用手机的WPS便签记笔记。
 
-**本节通过实操考察 pull request 的掌握情况，后面三节是 pull request 包含的内容**
+# 感想
 
-PR 考察分两阶段
-
-1. 第一阶段要求在11月7日21:00前开启你的 PR
-   - 值得注意的是，你不必在完成下面的内容后再开启 PR，PR 会自动跟踪你在自己分支上的修改
-   - 你的 PR 需要有合理且直观的标题和描述，不要空着，但这些内容是随时可以修改的，所以不必在开启 PR 时有过多的顾虑
-   - 你需要主动查看 PR 的合并检查项目并且解决自己可以解决的检查项目，下面介绍检查项目
-       - 你的 PR 需要至少一个 reviewer，如果你认为自己已经无事可做了，可以发送 request review 来使你的 PR 更快地合并
-       - 如果 reviewer 提出了修改意见，你需要在解决后回复，在确认完成后点击 resolved
-       - 你的修改历史必须是线性的，也就是不能出现分支间 merge
-       - 你增加的文件必须在 `2022-finalproject` 文件夹中
-2. 第二阶段要求在11月10日21:00前完成合并
-   - 值得注意的是，在第二阶段仍然可以修改或增加你的作业内容，但建议你尽量在这之前完成
-   - 我们会在两个阶段的临界点修改其中一个检查项目，你需要在这之后再次通过所有检查项目，通过后再次 request review，如果没有问题，你的 PR 就会被合并
-   - 被修改的检查项目为：作业提交文件夹变更为 `_2022-finalproject`
-   - 第二阶段不再接受新开的 PR，所以不要尝试通过重开 PR 或删除自己 Fork 的仓库来解决第二阶段提出的考验
-
-你可以在 MarkDown 文档中使用中文，在 PR 的描述及评论中使用中文，但**不要在 PR 的标题、commit 内容**中使用中文
-
-PR 的标题不是一个句子，请不要在末尾加上句号，并且需要用**极其精简且具有概括性的**语言来描述你的工作；在“描述”需要较为详细地讲述你完成的工作、修改的内容等
-
-你可以与同学讨论关于本节的内容，但后面的内容需要独立完成
-
-## 自述文件
-
-**要求：** 编写一个文档，对你自己在这门课程中的学习过程做一个总结，合理使用 MarkDown 的语法，字数不限，内容不限  
-**提交：** 一个文件，名称为 README.md
-
-## Git 应用
-
-**要求：** 编写一个文档，回答下列问题，这些问题的答案都应当使用 git 命令实现  
-**提交：** 一个 MarkDown 文件和若干个图片文件，在 MarkDown 中使用有效的`相对路径`引用你的图片
-
-1. 若你已经修改了部分文件、并且将其中的一部分加入了暂存区，应该如何回退这些修改，恢复到修改前最后一次提交的状态？给出至少两种不同的方式
-1. 若你已经提交了一个新版本，需要回退该版本，应该如何操作？分别给出不修改历史或修改历史的至少两种不同的方式
-1. 我们已经知道了合并分支可以使用 merge，但这不是唯一的方法，给出至少两种不同的合并分支的方式
+我是一名大一的小萌新，从高中开始就对机器人有着极大的热情，也对RM的赛事比较关注，在开学后了解到学校的对应的团队，我便报名加入了DynamicX团队的考核，但我对系统这方面简直是一窍不通，但通过团队这段时间的培训，我对系统的一些专有名词有所了解，并了解了对应的部分发挥怎么样的作用。即使现在的自己对很多东西都是一窍不通，但我相信，通过不断地学习，能够改变自己。但不管怎么说，我渴望拥有自己的一个机器人，渴望拥有一群机器人的伙伴，渴望拥有一支自己热爱的团队，渴望拥有一个为之奋斗的梦想。
```

Comment:
Please just don't modify this file, write it in your own README.md.

Reply:
Thanks,I have dealed with the problem you said